src/reanimated2/js-reanimated/MutableValue.js

```diff
@@ -0,0 +1,38 @@
+export default class MutableValue {
+  static MUTABLE_ID = 1;
+
+  _animation = null;
+  _listeners = [];
+
+  constructor(value, setter) {
+    this._id = MutableValue.MUTABLE_ID++;
+    this._value = value;
+    this._setter = setter;
+  }
+
+  get value() {
+    return this._value;
+  }
+
+  set value(nextValue) {
+    // eslint-disable-next-line
+    this._setter.apply(this, [nextValue]);
```

Comment:
Why this can't be just `this._setter(nextValue)`?

Reply:
Just wanted to make sure what `this` would be but in fact this straight call works as well.

Reply:
ef26915